src/components/Footer/DocsFooter.js

```diff
@@ -4,7 +4,10 @@ import { Row, Col } from 'antd'
 export function DocsFooter({ filename, title }) {
     return <div className='docs-footer'>
         <Row>
-            <h3>Help and feedback</h3>
+            <h3>Reach Out!</h3>
+            
+            If you need help on any of the above, feel free to create an issue on <a href='https://github.com/PostHog/posthog'>our repo</a>, or <a href='https://join.slack.com/t/posthogusers/shared_invite/enQtOTY0MzU5NjAwMDY3LTc2MWQ0OTZlNjhkODk3ZDI3NDVjMDE1YjgxY2I4ZjI4MzJhZmVmNjJkN2NmMGJmMzc2N2U3Yjc3ZjI5NGFlZDQ'>join our Slack</a> where a member of our team can assist you! Chances are that if you have a problem or question, someone else does too - so please don't hesitate to create a new issue or ask us a question :) <br /><br />
```

Comment:
Stylistically I'd remove the exclamation mark and the smiley at the end (sorry that's nitpicky!)

Reply:
totally understandable!